source/utils/types.ts

```diff
@@ -47,8 +52,16 @@ export interface InterfaceWithDefaults extends Instance {
 	};
 }
 
+interface RetryDescriptor {
+	// @todo Unsure of which types `retry` can be.
+	// Used `unknown` so anytime it'll be used, TS will complain and require a type assertion
+	// Maybe a maintainer can guide me here.
+	retries: (retry: unknown, error: Error) => number | number;
```

Comment:
My first guess is `retry` would be a `number` but I just needed some clarifications.

Reply:
You can just add a console.log to the code and check yourself whenever you wonder what the actual type is ;)

Reply:
`number | number;`?

Reply:
I've removed the comment and fixed the `retry` type to be `number`.
The whole type is now `() => number | number` which I believe is accurate now? Let me know if it's not.